src/components/SendLessonModal/SendLessonModal.stories.tsx

```
@@ -0,0 +1,411 @@
+        schoolId: 'escola-2',
+      },
+    ],
+    selectedIds: [],
+  },
+  {
+    key: 'turma',
+    label: 'Turma',
+    dependsOn: ['serie'],
+    filteredBy: [{ key: 'serie', internalField: 'schoolYearId' }],
+    itens: [
+      { id: 'turma-9a', name: '9º Ano A - Manhã', schoolYearId: 'serie-9-fund' },
```

ESLint

[error] Replace `·id:·'turma-9a',·name:·'9º·Ano·A·-·Manhã',·schoolYearId:·'serie-9-fund'` with `⏎········id:·'turma-9a',⏎········name:·'9º·Ano·A·-·Manhã',⏎········schoolYearId:·'serie-9-fund',⏎·····`

(prettier/prettier)